### PR TITLE
Upgraded double to boot-2

### PR DESCRIPTION
**Pull request by rudra-panda-dkl**
Please review and merge.

### CODE REVIEW

**Comment by pivotal-issuemaster** [comment]
@a759116 Please sign the [Contributor License Agreement](https://cla.pivotal.io/sign/spring?repositoryId=spring-guides/tut-spring-security-and-angular-js&pullRequestId=191)!

[Click here](https://cla.pivotal.io/sync/spring?repositoryId=spring-guides/tut-spring-security-and-angular-js&pullRequestId=191) to manually synchronize the status of this Pull Request.

See the [FAQ](https://cla.pivotal.io/about) for frequently asked questions.

**Comment by pivotal-issuemaster** [comment]
@a759116 Thank you for signing the [Contributor License Agreement](https://cla.pivotal.io/sign/spring?repositoryId=spring-guides/tut-spring-security-and-angular-js&pullRequestId=191)!

File: double/admin/pom.xml [hunk 1]
```diff
@@ -87,7 +87,7 @@
                     <arguments>run-script build</arguments>
                   </configuration>
                 </execution>
-                <execution>
+<!--                 <execution>
```
Comment (dsyer) [inline double/admin/pom.xml:90]: Why is this commented out? Isn't it needed to build the angular app?

File: double/admin/src/test/java/demo/ApplicationTests.java [hunk 2]
```diff
@@ -16,7 +15,7 @@
 @SpringBootTest(properties="security.user.password:foo", webEnvironment=WebEnvironment.RANDOM_PORT)
 public class ApplicationTests {
 
-	@LocalServerPort
+	@org.springframework.boot.web.server.LocalServerPort
```
Comment (dsyer) [inline double/admin/src/test/java/demo/ApplicationTests.java:18]: Please use an import.

File: double/gateway/pom.xml [hunk 3]
```diff
@@ -1,157 +1,157 @@
...
+					<execution>
+						<id>npm-build</id>
+						<goals>
+							<goal>npm</goal>
+						</goals>
+						<configuration>
+							<arguments>run-script build</arguments>
+						</configuration>
+					</execution>
+<!-- 					<execution>
```
Comment (dsyer) [inline double/gateway/pom.xml:117]: More commented out code?

File: double/gateway/src/main/java/demo/GatewayApplicationX.java [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+import org.springframework.security.core.authority.AuthorityUtils;
+import org.springframework.security.web.csrf.CookieCsrfTokenRepository;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.ResponseBody;
+
+@SpringBootApplication
+@Controller
+@EnableZuulProxy
+public class GatewayApplication {
```
Comment (dsyer) [inline double/gateway/src/main/java/demo/GatewayApplicationX.java:27]: This class is in a file with the wrong name?

File: double/resource/src/test/java/demo/ApplicationTests.java [hunk 5]
```diff
@@ -17,7 +16,7 @@
 @SpringBootTest(webEnvironment=WebEnvironment.RANDOM_PORT)
 public class ApplicationTests {
 
-	@LocalServerPort
+	@org.springframework.boot.web.server.LocalServerPort
```
Comment (dsyer) [inline double/resource/src/test/java/demo/ApplicationTests.java:19]: Use an import please.

File: double/ui/src/test/java/demo/ApplicationTests.java [hunk 6]
```diff
@@ -16,7 +15,7 @@
 @SpringBootTest(webEnvironment = WebEnvironment.RANDOM_PORT, properties = "security.user.password:foo")
 public class ApplicationTests {
 
-	@LocalServerPort
+	@org.springframework.boot.web.server.LocalServerPort
```
Comment (dsyer) [inline double/ui/src/test/java/demo/ApplicationTests.java:18]: Use an import please.

**Review by dsyer — COMMENTED** [review]
That's awesome, thanks. Made some comments.

**Pull request closed without merging**